src/test/java/web/tests/FlipkartTest.java

```diff
@@ -17,6 +20,20 @@ public void testSetUp() {
         Selenide.open(map.get("url"));
     }
 
+    /**
+     * testCurrentPage:selecting payment page and counting number of questions available on page.
+     */
+    @Test
+    public void testCurrentPage(){
+        PaymentPage paymentPage=new HomePage().popUpCancel().paymentPage();
+        String current_page_text=paymentPage.currentPageCheck();
+        logger.info(current_page_text);
+        softAssert.assertThat(current_page_text.equalsIgnoreCase("Payments"));
```

Comment:
use - =paymentPage.currentPageCheck() directly over here instead of using a new var - current_page_text